### DISCUSSION TITLE
혹시 매크로 돌리면서 다른 작업을 해도 괜찮나요?

### DISCUSSION THREAD

**Discussion by ghost** [changdoc/naver-vaccine-macro#49] [Q&A]
개발자님들 감사합니다 ㅎㅎ

혹시 매크로 돌리면서 롤 한판 해도 정상으로 작동이 되나요? 

아니면 롤과 백신중 하나만 선택해야만 하나요? ㅜㅜ

**Comment by max-jang** [comment]
회사에서 IDE 탭 두개 크롬 10개탭 정도에 엑셀 파워포인트 동시에 돌려도 느려지거나 하지는 않더라고요

**Comment by changdoc** [comment]
동작이 멈추지는 않아서 영향을 줄 것 같지는 않은데..
한번 시도 해보시죠.
